junit5/chapter.md

```diff
@@ -202,14 +203,16 @@ are involved. <br/>
 Taking the issues mentioned above and our initial stakeholder analysis into account, we created a context view of the
 system, as is visible in Fig. 3.
 
-![Context View](https://cloud.githubusercontent.com/assets/12841723/23343910/a747eaaa-fc73-11e6-93a1-24f27d5e90a5.png)
+![Context View](https://cloud.githubusercontent.com/assets/12841723/25781903/140581ea-3340-11e7-9d7e-64852c82c560.png)
```

Comment:
@avandeursen I believe you specifically instructed us to refrain from adding binaries to our git repositories, or should we remove this rubric from the SQTE course? 
![binaries](https://cloud.githubusercontent.com/assets/5991375/25782108/14bf4446-3344-11e7-82ab-d25adc7d6259.png)

(just joking)